Lib/test/test_turtle.py

```diff
@@ -1,5 +1,6 @@
 import pickle
 import unittest
+from unittest.mock import MagicMock, patch
```

Comment:
Let's use the same kind of imports as the other branch, it makes them easier to diff later:

```suggestion
import unittest
import unittest.mock
```

Reply:
Thats what i originally had, but tests were failing with that: https://github.com/python/cpython/actions/runs/19069447543/job/54468253248#step:9:674

Reply:
Not quite: looking at https://github.com/JanEricNitschke/cpython/commit/d19be3950067e7bce9bb9acacbe80c28ac4f4771 there was only `import unittest` before, not `import unittest` _and_ `import unittest.mock`.

This doesn't work:

```pycon
❯ python3.13
Python 3.13.6 (v3.13.6:4e665351082, Aug  6 2025, 11:22:35) [Clang 16.0.0 (clang-1600.0.26.6)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import unittest
>>> unittest.mock.MagicMock()
Traceback (most recent call last):
  File "<python-input-1>", line 1, in <module>
    unittest.mock.MagicMock()
    ^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/unittest/__init__.py", line 80, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'unittest' has no attribute 'mock'
>>>
```
But this does:
```pycon
❯ python3.13
Python 3.13.6 (v3.13.6:4e665351082, Aug  6 2025, 11:22:35) [Clang 16.0.0 (clang-1600.0.26.6)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import unittest
>>> import unittest.mock
>>> unittest.mock.MagicMock()
<MagicMock id='4320147648'>
>>>
```

Reply:
Ah, right, sorry. Didnt focus properly here. Accepted the suggestions now.